backend/app/api/v1/endpoints/cpt/cptJobs.py

```diff
@@ -156,20 +172,23 @@ async def filters(
         )
 
     filter_dict = get_dict_from_qs(filter) if filter else {}
+
     filter_product = filter_dict.pop("product", None)
 
-    if filter_product and filter_product not in productsFilter:
-        return Response(
-            content=json.dumps({"detail": "product not supported"}),
-            status_code=400,
-        )
+    individual_prod = ["ocp", "telco", "quay"]
```

Comment:
This is the second time this line has appeared -- `INDIVIDUAL_PROD` (or maybe something more like `PRODUCT_LIST`) `= ("ocp", "telco", "quay")` should be defined either here or maybe in `constants.py` rather than duplicating the literal.

Reply:
Moved the `individual_prod` list to the `constants.py` and renamed it.